deps/rabbit/src/rabbit_classic_queue.erl

```diff
@@ -569,3 +571,6 @@ send_credit_reply(Pid, QName, Len) when is_integer(Len) ->
         false ->
             gen_server2:cast(Pid, {send_credit_reply, Len})
     end.
+
+classic_queue_type_supports() ->
```

Comment:
Why an 's' to "supports"?

That's probably just me, but it makes it unclear what the function does.